routes/sites.js

```diff
@@ -1,25 +1,84 @@
 'use strict';
 
-// const pool = require('../config/config').pool;
+const Promise = require('bluebird');
+const query = require('../lib/utils').query;
 
+/**
+ * Gets all sites.
+ *
+ * @param {Object} req The given request object
+ * @return {Promise} The promise
+ */
 function getSites(req) {
+  if (req.query && req.query.coach_id) {
```

Comment:
Nice catch. I just asked someone on the front-end team. They don't need any `GET`ters like that.